components/daikin_s21/s21.cpp

```diff
@@ -543,7 +545,10 @@ void DaikinS21::handle_state_basic(std::span<uint8_t> &payload) {
     this->current.action_reported = daikin_to_climate_action(payload[1]);
   }
   this->current.climate.setpoint = (payload[2] - 28) * 5;  // Celsius * 10
-  // fan mode in payload[3], silent mode not reported so prefer RG
+  // silent fan mode not reported here so prefer RG if present
+  if (this->support.fan_mode_query == false) {
```

Comment:
@sstanfield I noticed on your unit the fan mode query RG isn't supported, this should pull it from the basic sensor readout G1.